### ISSUE TITLE
HabboParser.php error

### ISSUE THREAD

**Issue by ghost** [gerbenjacobs/HabboAPI#7]
Hey! I have a error....

ErrorException in HabboParser.php line 88:
Undefined index: error (View: /var/www/html/resources/views/page/startseite/startsite.blade.php)

Can anyone help me? Thanks!

**Comment by DavydeVries** [comment]
Can you send the following things?
- Request url
- Requested Habbo name/uuid (and country)

This make debugging a lot easier.

EDIT:
Probably you tried find a Habbo with one char. like "a". I received this by requesting "a":

```
{
    "errors": [
        {
            "param": "name",
            "msg": "user.invalid_name",
            "value": "a"
        }
    ]
}
```

The HabboParser.php code contains this (on line 88):

```
throw new Exception($data['error'],$data['habboAPI_info']['http_code']);
```

##### Conclusion:

The JSON key can contain `error` or `errors`.
So we need a solution for Undefined index error that you get.

##### ~~Quick and Dirty solution:~~ NOT WORKING

~~If you have program skills, if not you have to wait for the next update/release.~~

**Comment by DavydeVries** [comment]
I have created a basic `messageBag` based on the response `http_code`.
I going to send a pull request to gerberjacobs. (I think he will roll this out in the evening.)

**Comment by gerbenjacobs** [comment]
I have seen the error before. I believe there's one more case where there will be no `error` or `errors`. 
This could also happen if the hotel is down.

I've fixed it in this commit 20253d5a17482d643bd0d00b02545f238934e1f0

Maybe keep an eye out for other errors. (The problem of it being an unoffical API is that things can easily change without notice)
